### PR TITLE
Add case to cover initiator in iscsi hostdev

### PR DESCRIPTION
**Pull request by chunfuwen**
RHEL-196550:Enable initiator in iscsi hostdev

Signed-off-by: chunfuwen <chwen@redhat.com>

### CODE REVIEW

**Comment by chunfuwen** [comment]
DATA (filename=output.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stdout.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stderr.expected) => NOT FOUND (data sources: variant, test, file)
PASS 1-type_specific.io-github-autotest-libvirt.libvirt_scsi_hostdev.positive_test.single_dev.hot_plug.passthrough_iscsi_lun.enable_auth.sgio_default.rawio_default.not_shareable.not_readonly.managed_yes

**Comment by chunfuwen** [comment]
#https://github.com/avocado-framework/avocado-vt/pull/2947

**Comment by chunfuwen** [comment]
@lento-sun 
@kylazhang 

please help take a look

File: libvirt/tests/cfg/libvirt_scsi_hostdev.cfg [hunk 1]
```diff
@@ -11,6 +11,8 @@
                 - managed_yes:
                     only passthrough_iscsi_lun
                     managed = "yes"
+                    positive_test..single_dev..passthrough_iscsi_lun..enable_auth..sgio_default..rawio_default..not_shareable..not_readonly:
```
Comment (lento-sun) [inline libvirt/tests/cfg/libvirt_scsi_hostdev.cfg:14]: Hi Chunfu,
The newly added cases are:
libvirt.libvirt_scsi_hostdev.positive_test.single_dev.hot_plug.passthrough_iscsi_lun.enable_auth.sgio_default.rawio_default.not_shareable.not_readonly.managed_yes
and some other config branches.
We cannot figure out if the initiator iqn is set if we just read the patterns. Should we add it as a config variants such as:
```
- managed_yes:
    ...
    variants:
        - set_initiator_iqn:
             only ...
        - no_intiator_iqn:
```
And current setting will remove the cases which doesn't set the initiaor iqn.

Comment (chunfuwen) [reply libvirt/tests/cfg/libvirt_scsi_hostdev.cfg:14]: this will change case pattern if new variants is newly added

Comment (lento-sun) [reply libvirt/tests/cfg/libvirt_scsi_hostdev.cfg:14]: LGTM

**Review by lento-sun — APPROVED** [review]

File: libvirt/tests/src/libvirt_scsi_hostdev.py [hunk 2]
```diff
@@ -266,7 +268,9 @@ def check_disk_io(vm, partition):
     vm_name = params.get("main_vm")
     vm = env.get_vm(vm_name)
     virsh_dargs = {'debug': True, 'ignore_status': True}
-
+    enable_initiator_set = "yes" == params.get("enable_initiator_set")
```
Comment (kylazhang) [inline libvirt/tests/src/libvirt_scsi_hostdev.py:271]: better give default value with 'no' here

**Review by kylazhang — COMMENTED** [review]
Others are OK for me

File: libvirt/tests/src/libvirt_scsi_hostdev.py (hunk 2 above)
Comment (chunfuwen) [reply libvirt/tests/src/libvirt_scsi_hostdev.py]: updated

**Pull request merged**